cypress/integration/gov-swu-CR.spec.js

```diff
@@ -0,0 +1,183 @@
+        cy.get('#swu-opportunity-teaser').type('SWU cy teaser')
+        cy.get('#swu-opportunity-remote-ok-0').check({force:true})
+        cy.get('#swu-opportunity-remote-desc').type('SWU cy remote desc')
+        cy.get('#swu-opportunity-location').clear().type('Sechelt')
+        cy.get('#swu-opportunity-proposal-deadline').clear().type('2030-01-15')
+        cy.get('#swu-opportunity-assignment-date').clear().type('2030-01-31')
+        cy.get('#swu-opportunity-total-max-budget').clear().type('1000000')
+        cy.get('#swu-opportunity-min-team-members').clear().type('2')
+        cy.get('#swu-opportunity-mandatory-skills').type('Back-End Development{enter}')
+        cy.get('#swu-opportunity-mandatory-skills').type('Front-End Development{enter}')
+        cy.get('#swu-opportunity-optional-skills').type('Adobe Creative Cloud{enter}')
+        cy.get('a').contains('Next').click()
+
+        // 2. Description tab
+        cy.get('#swu-opportunity-description').type('SWU cy description')
+        cy.get('a').contains('Next').click()
+
+        // 3. Phases tab
+        cy.get('#swu-opportunity-starting-phase').click()
+        cy.get('#swu-opportunity-starting-phase').type('Inception{enter}')
+        // eslint-disable-next-line
+        cy.wait(1000) // Without this, something in the app resets the state and closes the drop-down, so can't fill out the fields
+        cy.get('div[class="h3 mb-0"]').contains('Inception').click()
+        cy.contains("During the Inception phase").should('be.visible')
+
+
+        cy.get('[id*=start-date]').first().type('2030-02-01')
+        cy.get('[id*=completion-date]').first().type('2030-02-28')
+        cy.get('[id*=max-budget]').first().type('200000')
+        cy.get('div[class="pt-2 pb-4 mb-4"]').first().contains('Delivery Management').click()
+        cy.contains('P/T').click()
+        cy.get('div[class="pt-2 pb-4 mb-4"]').first().contains('User Research').click()
+
+        cy.get('div[class="h3 mb-0"]').contains('Proof of Concept').click()
+        cy.contains("During the Proof of Concept phase").should('be.visible')
+        cy.get('[id*=start-date]').eq(1).type('2030-03-01')
+        cy.get('[id*=completion-date]').eq(1).type('2030-03-31')
+        cy.get('[id*=max-budget]').eq(1).type('200000')
+        cy.get('div[class="pt-2 pb-4 mb-4"]').eq(1).contains('Frontend Development').click()
+        cy.get('div[class="pt-2 pb-4 mb-4"]').eq(1).contains('Technical Architecture').click()
+
+        cy.get('div[class="h3 mb-0"]').contains('Implementation').click()
+        cy.contains("As you reach the Implementation phase").should('be.visible')
+        cy.get('[id*=start-date]').eq(2).type('2030-04-01')
+        cy.get('[id*=completion-date]').eq(2).type('2030-04-30')
+        cy.get('[id*=max-budget]').eq(2).type('400000')
+        cy.get('div[class="pt-2 pb-4 "]').contains('Security Engineering').click()
+        cy.get('div[class="pt-2 pb-4 "]').contains('Backend Development').click()
```

Comment:
This doesn't match above because the Implementation section doesn't have the same styles as the other two (thought visually they look the same)